src/demo/share/jfc/Font2DTest/Font2DTest.java

```diff
@@ -1021,17 +1429,23 @@ public static void main(String[] argv) {
             }
         }
 
-        UIManager.put("swing.boldMetal", Boolean.FALSE);
-        final JFrame f = new JFrame( "Font2DTest" );
-        final Font2DTest f2dt = new Font2DTest( f);
-        f.addWindowListener( new WindowAdapter() {
-            public void windowOpening( WindowEvent e ) { f2dt.repaint(); }
-            public void windowClosing( WindowEvent e ) { System.exit(0); }
-        });
-
-        f.getContentPane().add( f2dt );
-        f.pack();
-        f.setVisible(true);
+        buildFontInfo();
+        try {
+           UIManager.setLookAndFeel(new NimbusLookAndFeel());
+            SwingUtilities.invokeAndWait(() -> {
+                final JFrame f = new JFrame( "Font2DTest" );
+                final Font2DTest f2dt = new Font2DTest( f);
+                f.addWindowListener( new WindowAdapter() {
+                    public void windowOpening( WindowEvent e ) { f2dt.repaint(); }
+                    public void windowClosing( WindowEvent e ) { System.exit(0); }
+                });
+
+                f.getContentPane().add( f2dt );
```

Comment:
Since this is new code, we can remove getContentPane which is not needed for add